compiler/rustc_hir/src/lang_items.rs

```diff
@@ -139,6 +139,12 @@ pub fn extract(attrs: &[ast::Attribute]) -> Option<(Symbol, Span)> {
 
 language_item_table! {
 //  Variant name,            Name,                     Getter method name,         Target                  Generic requirements;
+    ///  reserved lang items for `default_auto_traits` feature
+    Trait4,                  sym::trait4,              trait4_trait,               Target::Trait,          GenericRequirement::None;
```

Comment:
```suggestion
    DefaultTrait4,                  sym::default_trait4,              default_trait4,               Target::Trait,          GenericRequirement::None;
```

Reply:
Also makes sense to move these to the end of the lang feature list, so it doesn't *immediately* confuse people reading the code.